### PR TITLE
[x/debug] Fix panic when placement is nil

### PR DESCRIPTION
**Pull request by vpranckaitis**
**What this PR does / why we need it**:

Fixes a panic in debug handler caused by `nil` placement. 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing and/or backwards incompatible change?**:

```release-note
NONE
```

**Does this PR require updating code package or user-facing documentation?**:

```documentation-note
NONE
```

### CODE REVIEW

File: src/x/debug/ext/placement.go [hunk 1]
```diff
@@ -60,6 +58,13 @@ func (p *placementInfoSource) Write(w io.Writer, httpReq *http.Request) error {
 		return err
 	}
 
+	if placement == nil {
+		if _, err := w.Write([]byte("{}")); err != nil {
```
[the remark this answers is not in the document]
Comment (linasm) [reply src/x/debug/ext/placement.go:62]: Shouldn't we return an error instead?

[the remark this answers is not in the document]
Comment (vpranckaitis) [reply src/x/debug/ext/placement.go]: https://github.com/m3db/m3/pull/4046/commits/1cd2b87be7a736c71f4c48a6cfcf0fee10d860bf

**Review by Antanukas — APPROVED** [review]
LGTM.

**Review by linasm — APPROVED** [review]
LGTM

**Pull request merged**